legend-engine-pure/legend-engine-pure-code/legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/preeval/preeval.pure

```diff
@@ -1087,4 +1089,4 @@ function <<access.private>> meta::pure::router::preeval::printDebug(debugContext
     | print(''),
     | println($param);
   );
-}
```

Comment:
Is it possible to add a unit test for this? thanks

Reply:
will reopen